Revised files to accommodate a new learning algorithm and to run simulations in parallel.

xyzeroth opened the pull request:
수정된 파일(또는 디렉토리) 이름에 'off_ppo'를 추가하여, 기존 코드와 구별되도록 하였습니다.

'off-ppo'로 디렉토리를 만들었다가, 'off_ppo'로 변경하여 다시 파일을 올렸습니다.

디렉토리 이름만 'off-ppo'에서 'off_ppo'로 변경한 것 이기 때문에, 'off-ppo'로 되어 있는 부분은 무시하셔도 됩니다.

xyzeroth committed:
Revised the files, originally located in 'env', to accommodate a new learning algorithm.

xyzeroth committed:
Add files via upload

xyzeroth committed:
Update reame.txt

xyzeroth committed:
Add files via upload

xyzeroth committed:
Add files via upload

xyzeroth committed:
Add files via upload

xyzeroth committed:
Delete atsc-rl/multiagent_tf2/env/off-ppo directory

xyzeroth committed:
Revised the files, originally located in 'env', to accommodate a new learning algorithm.

xyzeroth committed:
Add files via upload

xyzeroth committed:
Add files via upload

xyzeroth committed:
Add files via upload

hunsooni commented:
TSOConstants.py와 TSOConstants_off_ppo.py, TSOUtil.py, TSOUtil_off_ppo.py는 하나로 합치는 것이 좋을 것 같습니다.

xyzeroth committed:
Add files via upload

TSOConstants_off_ppo.py,   TSOUtil_off_ppo 대신에 TSOConstants와 TSOUtil를 import 하는 것으로 변경하였습니다.

xyzeroth committed:
Add files via upload

출력 경로를 getOutputDirectoryRoot ()에서 가지고 오도록 수정.

xyzeroth committed:
Add files via upload

Line 162: def addArgumentsToParser(parser):
Default 값은 편하신대로 설정하여 사용하시기 바랍니다.

Line 182  parser.add_argument('--scenario', choices=['12th', 'kaist' ], default='12th', help='simulation scenario').
현재 실험을 위한 것으로, 당분간은 남겨 두는 것이 좋을 것 같습니다.

xyzeroth committed:
Add files via upload

xyzeroth committed:
Add files via upload

xyzeroth committed:
Add files via upload

시뮬레이션 결과 파일이 저장되는 경로는, 시뮬레이터(libsalt) 쪽에서 hard coding 되어 있는 것으로 알고 있습니다.
확인해 보시고, testSappo()에서 아래의 args.io_home에 해당되는 경로는 직접 수정하시는 것이 좋을 것 같습니다.
        ft_output = pd.read_csv("{}/output/simulate/{}/{}".format(args.io_home, args.scenario, _RESULT_COMP_.SIMULATION_OUTPUT))
        rl_output = pd.read_csv("{}/output/test/{}/{}".format(args.io_home, args.scenario, _RESULT_COMP_.SIMULATION_OUTPUT))
